packages/ui-extensions/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@shopify/ui-extensions",
   "version": "0.0.0-unstable",
   "scripts": {
-    "docs:admin": "sh ./docs/surfaces/admin/build-docs.sh \"admin\"",
```

Comment:
Thanks for cleaning this up lol, that was from an experiment I was doing before summer editions